explorer/app/viewModelBuilders/azul/anvil-cmg/common/viewModelBuilders.ts

```diff
@@ -217,6 +217,19 @@ export const buildDataReleasePolicy = (): React.ComponentProps<
   };
 };
 
+/**
+ * Build dataset access badge component from the given index/datasets response.
+ * @param response - Response model return from index/datasets API.
+ * @returns model to be used as props for the dataset access cell.
+ */
+export const buildDatasetAccess = (
+  response: DatasetsResponse
+): React.ComponentProps<typeof C.AccessStatusBadge> => {
+  return {
+    accessible: response.datasets[0].accessible,
```

Comment:
This is only a suggestion: take a look at the method `processEntityValue` from https://github.com/clevercanary/data-browser/blob/9e9a2e7345ecca0854b9310e563dca003590ee09/explorer/app/apis/azul/common/utils.ts and return LABEL.EMPTY if the value is undefined - and you can then convert to `false` with an `|| false` on the returned value. Or jut leave this as is!

Reply:
Urgh @hunterckx good pick up - `processEntityValue` is for strings 🙄.

Let's leave this as is for now!